src/components/cell-control/input.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import CellControl from '../cell-control/cell-control.jsx';
+import InputControl from '../control/input.jsx';
+
+function Input(props) {
+  return (
+    <CellControl
+      labelledBy={props.labelledBy}
+      readOnly={props.readOnly}
+    >
+      <InputControl
+        id={props.id}
+        labelledBy={props.labelledBy}
+        readOnly={props.readOnly}
+        required={props.required}
+        validationMessage={props.validationMessage}
+      />
+    </CellControl>
+  );
+}
+
+Input.propTypes = {
+  /** A unique ID for the component. */
```

Comment:
I'm really liking inline documentation... But I can see how this can be repeated content. 🤔